src/main/app/pdf/responseReceipt.ts

```diff
@@ -30,7 +30,7 @@ export class ResponseReceipt {
         this.claim.claimantEmail
       ),
       defendant: DefendantMapper.createDefendantDetails(
-        this.claim.response.defendant,
+        this.claim.response.defendantDetails,
```

Comment:
Please make sure that does not make master otherwise we would revert @radoslawn-kainos change.